src/map/status.h

```diff
@@ -2074,14 +2074,15 @@ struct status_data {
 		hp, sp,  // see status_cpy before adding members before hp and sp
 		max_hp, max_sp;
 	short
-		str, agi, vit, int_, dex, luk;
-	unsigned short
-		batk,
+		str, agi, vit, int_, dex, luk,
+		batk;
```

Comment:
I didn't really touch other stats, but let me test if it'll have any adverse effect when used as negative bonuses